### PR TITLE
feat/provision with tools

### PR DESCRIPTION
**Pull request by marcellmueller**
This is branched off of [the chart releaser workflow ](https://github.com/bcgov/crunchy-postgres/pull/21).

-Moved all of the networking and deployer templates into tools and deploy using a handy script I borrowed from (cas pipeline)[https://github.com/bcgov/cas-pipeline].

### CODE REVIEW

File: charts/tools/values.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+fullnameOverride: crunchy-postgres-tools
```
Comment (dleard) [inline charts/tools/values.yaml:1]: Is this the example values.yaml file that defines the secret values file?

**Review by dleard — CHANGES_REQUESTED** [review]
A couple version updates and a question about values files, but otherwise this looks great!

File: charts/tools/values.yaml (hunk 1 above)
Comment (marcellmueller) [reply charts/tools/values.yaml:1]: Yes it is, do you think I should use a more generic name for the default?

File: .github/workflows/release-chart.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.1
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply .github/workflows/release-chart.yaml]: Nice catch, I updated these thanks!

File: charts/tools/values.yaml (hunk 1 above)
Comment (dleard) [reply charts/tools/values.yaml:1]: I guess it just wasn't clear to me that this was an example file. A comment at the top to make this clearer would help. And maybe an example for the networking.route.host value

Comment (marcellmueller) [reply charts/tools/values.yaml:1]: I'm a bit confused to be honest, this is the `values.yaml` that defines all of the  expected values. Isn't that standard with helm charts?

**Pull request closed without merging**